docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -1,9 +1,38 @@
 
+tag::allow-no-indices[]
+`allow_no_indices`::
+(Optional, boolean) If `true`, the request returns an error if a wildcard
+expression or `_all` value retrieves only missing or closed indices. This
+parameter also applies to <<indices-aliases,index aliases>> that point to a
+missing or closed index.
+end::allow-no-indices[]
+
 tag::bytes[]
 `bytes`::
 (Optional, <<byte-units,byte size units>>) Unit used to display byte values.
 end::bytes[]
 
+tag::expand-wildcards[]
+`expand_wildcards`::
++
+--
+(Optional, string) Controls what kind of indices that wildcard
+expressions can expand to. Valid values are:
+
+`open`::
```

Comment:
Should it be `open,closed` rather than simply `open`? The next value is also `open`, based on the description the first one should be changed.

Reply:
Thanks for catching that. It should be `all`. Fixed with b8653d3.